board/piksiv3/linux.config

```diff
@@ -61,8 +61,14 @@ CONFIG_IP_PNP_RARP=y
 CONFIG_NET_IPIP=m
 CONFIG_VLAN_8021Q=m
 CONFIG_CAN=y
+CONFIG_CAN_DEV=y
+CONFIG_CAN_CALC_BITTIMING=y
+CONFIG_CAN_RAW=y
 CONFIG_CAN_XILINXCAN=y
 CONFIG_CAN_DEBUG_DEVICES=y
+CONFIG_CAN_C_CAN_PLATFORM=y
```

Comment:
These are just lucky-tries to get the `can_*` stuff to work, most likely unnecessary.

Reply:
Most of these are most likely not needed. I need someone with access to debug devices to test which ones are necessary.